lib/go/payloads.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+type SignupRequestPayload struct {
```

Comment:
I know it's more tedious to write it manually _but_ we'll probably end up generating this code so it's better to move the payloads to their own package, one payload per file